packages/lambda-term/lambda-term.3.3.2/opam

```diff
@@ -0,0 +1,49 @@
+opam-version: "2.0"
+synopsis: "Terminal manipulation library for OCaml"
+description: """
+Lambda-term is a cross-platform library for manipulating the terminal. It
+provides an abstraction for keys, mouse events, colors, as well as a set of
+widgets to write curses-like applications. The main objective of lambda-term is
+to provide a higher level functional interface to terminal manipulation than,
+for example, ncurses, by providing a native OCaml interface instead of bindings
+to a C library. Lambda-term integrates with zed to provide text edition
+facilities in console applications."""
+maintainer: ["opam-devel@lists.ocaml.org"]
```

Comment:
This email address is for the opam developers, who are not maintaining this package. Is there any way to put yourself there instead?

Reply:
That would be fine :)